src/main/java/com/example/waggle/global/util/ObjectUtil.java

```diff
@@ -15,7 +15,10 @@ public static boolean isPresent(@Nullable Object object) {
     }
 
     public static void validateKeywordLength(String keyword) {
-        if (keyword == null || keyword.length() < 2) {
+        if (keyword == null) {
+            return;
+        }
+        if (keyword.length() < 2) {
```

Comment:
키워드 최소 길이 상수로 관리하면 보기 좋을 것 같아요!
```suggestion
private static final int MIN_KEYWORD_LENGTH = 2;

public static void validateKeywordLength(String keyword) {
    if (ObjectUtils.isEmpty(keyword) || keyword.length() < MIN_KEYWORD_LENGTH) {
        throw new GeneralException(ErrorStatus.BOARD_SEARCHING_KEYWORD_IS_TOO_SHORT);
    }
}
```